android-ktx/src/main/java/com/mirego/trikot/viewModels/ImageViewModelBinder.kt

```diff
@@ -130,6 +125,7 @@ object ImageViewModelBinder {
                 })
             } ?: run {
                 imageFlow.placeholderImageResource?.asDrawable(imageView.context)?.let {
+                    placeholderScaleType?.let { imageView.scaleType = it }
```

Comment:
What happens when users sets a `placeholderScaleType` but does not set an ScaleType ? Would the default ScaleType value remains or the placeholderScaleType will be applied to the Image ?

On iOS, we "save" the current contentMode and set it back once the image is loaded.

Please ignore this comment if saving content mode is not required.

Reply:
That wasn't required since the use case we supported previously only covered the case where no url and no image resource was provided.

However, I added support for the case where url is provided and that is covered now.